requirements.txt

```
@@ -3,7 +3,7 @@ asteval==0.9.27
 Babel==2.11.0
 click==8.1.3
 contourpy==1.0.6
-CoolProp==6.4.1
+CoolProp==6.4.2.dev0
```

Its out now so we can just use it directly
```suggestion
CoolProp==6.4.2
```